src/Simplex/FileTransfer/Agent.hs

```diff
@@ -398,15 +399,11 @@ runXFTPSndPrepareWorker c doWork = do
           chunkDigests <- map FileDigest <$> mapM (liftIO . getChunkDigest) chunkSpecs
           pure (FileDigest digest, zip chunkSpecs chunkDigests)
         createChunk :: Int -> SndFileChunk -> m ()
-        createChunk numRecipients' SndFileChunk {sndChunkId, userId, chunkSpec = XFTPChunkSpec {chunkSize}, digest = FileDigest chDigest} = do
-          (sndKey, spKey) <- liftIO $ C.generateSignatureKeyPair C.SEd25519
-          rKeys <- liftIO $ L.fromList <$> replicateM numRecipients' (C.generateSignatureKeyPair C.SEd25519)
-          let fileInfo = FileInfo {sndKey, size = fromIntegral chunkSize, digest = chDigest}
-          srvAuth@(ProtoServerWithAuth srv _) <- getServer
-          (sndId, rIds) <- agentXFTPCreateChunk c userId srvAuth spKey fileInfo (L.map fst rKeys)
-          let rcvIdsKeys = L.toList $ L.map ChunkReplicaId rIds `L.zip` L.map snd rKeys
-          withStore' c $ \db -> createSndFileReplica db sndChunkId srv (ChunkReplicaId sndId) spKey rcvIdsKeys
-          addXFTPSndWorker c $ Just srv
+        createChunk numRecipients' ch = do
+          srv@(ProtoServerWithAuth server _) <- getServer
```

Comment:
`srvAuth@(ProtoServerWithAuth srv _)` I actually copied naming from smp, I'd keep as it was as it's easier to search for same occurence, also the difference when calling functions is clearer